CONTRIBUTING.md

```diff
@@ -139,7 +139,16 @@ A guide on commonly used labels added to issues and pull requests:
 
 ## Testing Your Code
 
-- You can test your code locally along with your changes using: `yarn storybook` or `npm run storybook`. This will open the storybook in your browser .
+- You can test your code locally along with your changes using: `yarn storybook` or `npm run storybook`. This will open the storybook in your browser.
+
+This project contains Jest tests. These tests can be run using `yarn test`. When new features are added, a Jest test should also be added. The following best practices should be observed when writing Jest tests:
```

Comment:
...`yarn test` or `npm run test`

Reply:
"When new features are added..." -> "When changes to functionality are made, tests should always be included". Something along those lines could be more clear.

Reply:
Maybe to be more clear about testing and what technologies are used ... "This project contains test that are executed using `jest`. The bulk of the component tests are written with React Testing Library..."

Reply:
Good comments! I included these in the latest commit and made it more clear that I was referring to react testing library